samples/Sentry.Samples.AspNetCore.Grpc/Program.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Net;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+
+namespace Sentry.Samples.AspNetCore.Grpc
+{
+    public static class Program
+    {
+        public static void Main(string[] args)
+        {
+            BuildWebHost(args).Run();
+        }
+
+        public static IWebHost BuildWebHost(string[] args) =>
+            WebHost.CreateDefaultBuilder(args)
+                .UseShutdownTimeout(TimeSpan.FromSeconds(10))
+                .UseStartup<Startup>()
+
+                // Example integration with advanced configuration scenarios:
+                .UseSentryGrpc(options =>
```

Comment:
I'm not sure I follow the approach here. If I had a normal ASP.NET Core app, now I'd need to change my UseSentry to be UseSentryGrpc? 

Couldn't se hook through the container? Basically registering a `EventProcessor` so that the plain ASP.NET Core integration can pick it up?

Reply:
I suppose my initial approach was more under the assumption that it would be an either-or sort of thing.

The issue with just making it an `EventProcessor` would be that the plain ASP.NET Core integration does not catch the exceptions thrown by gRPC services. I'm not really sure why that is the case, but it seems to be by design (refer to https://docs.microsoft.com/en-us/aspnet/core/grpc/migration?view=aspnetcore-3.1#grpc-interceptors-vs-middleware).

Reply:
I see. Good to know, thanks for clarifying!